addons/im_livechat/static/tests/helpers/mock_models.js

```diff
@@ -26,7 +26,7 @@ MockModels.patch('im_livechat/static/tests/helpers/mock_models.js', T =>
             Object.assign(data['mail.channel'].fields, {
                 anonymous_name: { string: "Anonymous Name", type: 'char' },
                 country_id: { string: "Country", type: 'many2one', relation: 'res.country' },
-                livechat_active: { string: "Is livechat ongoing?", type: 'boolean', default: false },
+                livechat_active: { string: "Is livechat ongoing?", type: 'boolean', default: true },
```

Comment:
keep false, make true in the test that need it.

Reply:
Reasoning: value in mock server should be the same as in the server. And server has default false. Make sure to update the mock server that create a livechat to set the value to true if needed.

Also don't forget to add a test for this new feature.